images/airflow/2.10.3/python/mwaa/database/migrate.py

```diff
@@ -40,12 +40,22 @@ def _migrate_db():
         args = Namespace(migration_wait_timeout=1)
         airflow_db_command.check_migrations(args)
         logging.info("The database is already migrated.")
+        args = Namespace(
```

Comment:
The addition of these args here isn't present in the 2.10.1 and 2.9.2 changes. Is this intentional?

Reply:
Yes that should be removed, I added that to test and it thought I deleted it